{{cookiecutter.github_repo_name}}/bin/submodules_processor.py

```diff
@@ -16,8 +16,11 @@
     repo = Repo()
     for submodule in repo.iter_submodules():
         if submodule.url.startswith("git@"):
+            # All github SSH repos are like git@github.com:<repo_name>.git
+            starting_index = submodule.url.find(":") + 1
```

Comment:
If you use `split` you can keep the substring with `repo_infos_str = submodule.url.split(':')[-1]`

Reply:
Anyway, the result is the same, and with the `find()` it works as well in the case of repos that may contain `:` after `git@github.com:` (don't know if it's possible this situation)